build/docker/agent/Dockerfile

```diff
@@ -165,6 +165,9 @@ COPY --chown=$USERNAME:$USERNAME ./code /workspace/code/
 # Link code into catkin workspace
 RUN ln -s /workspace/code /catkin_ws/src
 
+# For debugger
+ENV PAF_CATKIN_CODE_ROOT=/catkin_ws/src
+
```

Comment:
Consider changing the sequence such that you use your new env variable in linking the code to the catkin_ws